### DISCUSSION TITLE
Having trouble running the project

### DISCUSSION THREAD

**Discussion by avineas** [deckerst/aves#1244] [Q&A] [answered]
Cloned the repo on windows 10, ran `scripts/apply_flavor_izzy.sh` then ran `./flutterw run -t lib/main_izzy.dart --flavor izzy` which gives me this error: 

> Execution failed for task ':aves_screen_state:compileDebugKotlin'.
> Inconsistent JVM-target compatibility detected for tasks 'compileDebugJavaWithJavac' (21) and 'compileDebugKotlin' (17). Consider using JVM Toolchain: https://kotl.in/gradle/jvm/toolchain

**Comment by deckerst** [comment, marked as answer]
I've reviewed the build settings. Can you update your repo and try again?
